devdoc/channel.md

```diff
@@ -310,3 +317,28 @@ channel_open` opens the given `channel`.
 **SRS_CHANNEL_43_157: [** `execute_callbacks` shall call `sm_exec_end` for each callback that is called. **]**
 
 **SRS_CHANNEL_43_147: [** `execute_callbacks` shall perform cleanup of the `operation`. **]**
+
+### channel_get_count_of_items_in_channel
```

Comment:
>channel_get_count_of_items_in_channel [](http://example.com/codeflow?start=4&length=37)

Consider adding "volatile" to the name of the channel to indicate that the count is very transient.

Reply:
new iteration has more data and the new API is called sealable_channel_get_reporting_info since it's for reporting only.  (and some verification tests).